src/GitHook/Command/FileCommand/PreCommit/FrontendFormatterCommand.php

```diff
@@ -42,7 +42,7 @@ public function run(CommandContextInterface $context): CommandResultInterface
     {
         $commandResult = new CommandResult();
 
-        $processDefinition = ['node', './frontend/libs/formatter', '--fix', '--ignore-path', './.prettierignore-githook', '--file-path', $context->getFile()];
+        $processDefinition = ['npx', 'prettier', $context->getFile(), '--check', '--ignore-path', './.prettierignore-githook'];
```

Comment:
* We have to align with FE team that this is expected behavior change.

Reply:
yes, it was confirmed with FE developer.